packages/rn-tester/e2e/test/steps/buttonComponentScreen.steps.js

```diff
@@ -0,0 +1,45 @@
+import componentsScreen from '../screenObjects/components.screen.js';
+import buttonComponentScreen from '../screenObjects/buttonComponent.screen.js';
+import { expect } from '@jest/globals';
+
+export const thenVerifyThatTheButtonComponentIsDisplayed = (then) => {
+    then(/^Verify that the Button component is displayed$/, async () => {
+        expect(await componentsScreen.checkButtonComponentIsDisplayed()).toBeTruthy();
+    });
+};
+
+export const whenUserClicksOnTheButtonComponent = (when) => {
+    when(/^User clicks on the Button component$/, async () => {
+        await componentsScreen.clickButtonComponent();
+    });
+};
+
+export const thenVerifyThatTheButtonHeaderIsDisplayed = (then) => {
+    then(/^Verify that the "(.*)" header is displayed$/, async (headerScreenName) => {
+        expect(await buttonComponentScreen.checkButtonsScreenIsDisplayed()).toContain(headerScreenName);
+    });
+};
+
+export const whenUserClicksOnTheCancelApplicationButton = (when) => {
+    when(/^User clicks on the Cancel Application button$/, async () => {
+        await buttonComponentScreen.clickCancelApplication();
+    });
+};
+
+export const thenVerifyAlertBoxHasText = (then) => {
+    then(/^Verify that the alert box has text: "(.*)"$/, async (alertBoxText) => {
+        expect(await buttonComponentScreen.getCancelAlertText()).toContain(alertBoxText);
+    });
+};
+
+export const whenUserClicksOnTheOKButton = (when) => {
+    when(/^User clicks on the OK button$/, async () => {
+        await buttonComponentScreen.clickOkButton();
+    });
+};
+
+export const whenUserClicksOnTheSubmitApplicationButton = (when) => {
+    when(/^User clicks on the Submit Application button$/, async () => {
```

Comment:
this could have been a regular string instead of a regex, right?

Reply:
I believe we should keep the consistency of the steps format

Reply:
I see a regex rather an exception from the rule of using strings. There's nothing wrong with having a little bit of inconsistency here. Personally it would be just harder for me to write a working regex than a plain string. Using a regex also triggers a different mental path for me (my mind goes "whats weird about this code that it uses regex instead of a string?" – if it finds nothing weird, then it's the time that I could save by using a faster string lookup that I'd normally expect). Again, I'm speaking for my own. If you find this is saving you time and no one else complains, please go with it

Reply:
I have checked couple of repos to verify how it is resolved there. Most of them use consistent regex format for every step. But we are fine with both approaches. I understand your point of view. I have changed steps to the expected format. Thanks! c: